src/components/Ticket/Ticket.tsx

```diff
@@ -0,0 +1,230 @@
+import { Box, Stack, Typography, useMediaQuery } from '@mui/material';
+import DirectionsBusRoundedIcon from '@mui/icons-material/DirectionsBusRounded';
+import { TicketWrapper } from './Ticket.styled';
+import Barcode from 'react-barcode';
+import { TwoLineHeading } from './components/TwoLineHeading';
+import { ITicket } from '../../types';
+import { formatDate } from './utils/timeUtils';
+
+export const Ticket = ({ data }: { data: ITicket }) => {
+    const isSmallScreen = useMediaQuery('(max-width:860px)');
+    const isMediumScreen = useMediaQuery('(max-width:1024px)');
+
+    const { pnrNumber, trip, seats } = data;
+    const { departureTimestamp, arrivalTimestamp } = trip;
+
+    const adults = seats.filter((seat) => seat.passenger.age > 18);
+    const children = seats.filter((seat) => seat.passenger.age < 18);
+
+    return (
+        <TicketWrapper>
+            <Box>
+                <Stack direction={'row'} justifyContent={'space-between'}>
+                    <DirectionsBusRoundedIcon
+                        fontSize="medium"
+                        className="app-logo"
+                        color="secondary"
+                    />
+                    <Typography component={'h1'} variant="h2">
+                        TICKET
```

Comment:
give translation to TICKET

Reply:
Resolved in 75a1ed6dca81a82813efc1f53acc0e96bd6374d4